cluster/gce/config-default.sh

```diff
@@ -567,3 +567,7 @@ export WINDOWS_ENABLE_HYPERV="${WINDOWS_ENABLE_HYPERV:-false}"
 # TLS_CIPHER_SUITES defines cipher suites allowed to be used by kube-apiserver.
 # If this variable is unset or empty, kube-apiserver will allow its default set of cipher suites.
 export TLS_CIPHER_SUITES=""
+
+# CLOUD_PROVIDER defines the cloud-provider value presented to KCM, apiserver,
+# and kubelet
+export CLOUD_PROVIDER="${CLOUD_PROVIDER:-gce}"
```

Comment:
The CLOUD_PROVIDER variable is already used [here](https://github.com/kubernetes/kubernetes/blob/master/cluster/kubemark/util.sh#L19). I would suggest switching this to something like CLOUD_PROVIDER_FLAG. Its more descriptive and not yet used.